http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/http2/Http2MultiplexedChannelPool.java

```diff
@@ -316,7 +316,7 @@ private boolean acquireStreamOnInitializedConnection(MultiplexedChannelRecord ch
     }
 
     public void handleGoAway(Channel parentChannel, int lastStreamId, GoAwayException exception) {
-        log.debug(() -> "Received GOAWAY on " + parentChannel + " with lastStreamId of " + lastStreamId);
+        log.warn(() -> "Received GOAWAY on " + parentChannel + " with lastStreamId of " + lastStreamId);
```

Comment:
Still not sure if we should set it to `warn` as `GOAWAY` is not uncommon and retry normally should work. Using `warn` might raise unnecessary concerns. 

Can we add the message in the exception instead?

Reply:
Yeah looking at this again, I don't think it's necessary to increase the level here. The changes `MultiplexedChannelRecord` should be all we need